index.html

```diff
@@ -0,0 +1,142 @@
+
+                    </a>
+                </li>
+                <li class="aside-4 a-li">
+                    <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/supplier-responsibility/_1">
+
+                    </a>
+                </li>
+            </ul>
+        </aside>
+
+
+        <footer>
+            <div class="footer-div">
+                <p class="footer-p">Shop the <a href="https://web.archive.org/web/20140301004610/http://store.apple.com/us">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="https://web.archive.org/web/20140301120134/http://www.apple.com/retail/">Apple Retail Store</a>, or find a <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/buy/_1">reseller</a>.</p>
+                <ul>
+                    <li class="footer-lis">
+                        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/sitemap/_1">Site Map </a>
+                    </li>
+                    <li>|</li>
+                    <li class="footer-lis">
+                        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/hotnews/_1">Hot News </a>
+                    </li>
+                    <li>|</li>
+                    <li class="footer-lis">
+                        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/rss/_1">RSS Feeds </a>
+                    </li>
+                    <li>|</li>
+                    <li class="footer-lis">
+                        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/pr/_1">Media Info </a>
+                    </li>
+                    <li>|</li>
+                    <li class="footer-lis">
+                        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/environment/_1">Environment </a>
+                    </li>
+                    <li>|</li>
+                    <li class="footer-lis">
+                        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/jobs/_1">Job Opportunities </a>
+                    </li>
+                    <li>|</li>
+                    <li class="footer-lis">
+                        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/contact/_1">Contact Us </a>
+                    </li>
+                    <li>|</li>
+                    <li class="footer-lis">
+                        <a class="flag" href="https://web.archive.org/web/20140301004610/http://www.apple.com/choose-your-country/_1">
+
+                        </a>
+                    </li>
+                </ul>
+                <p class="footer-p">Copyright © 2014 Apple Inc. All rights reserved. <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/_1">Terms of Use</a> | <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/privacy/_1">Privacy Policy</a> </p>
+            </div>
+        </footer>
+    </main>
```

Comment:
> The `<main>` element is used to denote the content of a webpage that relates to the central topic of that page or application. It should include content that is unique to that page and should not include content that is duplicated across multiple webpages, such as headers, footers, and primary navigation elements.

[Read more](https://html.com/tags/main/#ixzz5yaUVSP20)

Reply:
Yes you are right, it was my fault, i was careless a little bit but thanks for comments. 
I've fixed both problems now ^^